src/background/listener/bee-api.listener.ts

```diff
@@ -0,0 +1,85 @@
+import { StoreObserver, getItem } from '../../utils/storage'
+export class BeeApiListener {
+  private beeApiUrl: string
+
+  public constructor(private storeObserver: StoreObserver) {
+    this.beeApiUrl = 'http://localhost:1633'
+    this.addStoreListeners()
+    this.asyncInit()
+    this.addBzzListeners()
+  }
+
+  private addBzzListeners() {
+    // typed in 'bzz://{content-address}' URI in search bar triggers redirect to gateway BZZ address
+    // only react if google search has been set for searching
```

Comment:
Do I understand this right? It it really search bar or should it be address bar?

```suggestion
    // 'bzz://{content-address}' URI in search bar triggers redirect to gateway BZZ address
    // NOTE: works only if google search is set as default search engine
```